{"name": "Проверка отображения существующего заказа из истории пользователя в ленте заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Не удалось найти элемент ['xpath', \".//h1[contains(text(), 'Соберите бургер')]\"]\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:527:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "browser = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"4659ec91-39bf-41e5-96c9-872944db68ab\")>\ncreate_user = ({'email': 'xcollins@example.net', 'name': 'Stephanie Christensen', 'password': 'B#2Tux37#9'}, <Response [403]>)\n\n    @pytest.fixture\n    def login_random_user(browser, create_user):\n        \"\"\"\n        Фикстура для логина в систему с помощью данных созданного пользователя.\n        \"\"\"\n        create_user_data = create_user[0]  # Получение данных созданного пользователя\n        main_page = MainPage(browser, BASE_URL)  # Инициализация страницы с заголовком\n        main_page.click_user_account_button()  # Клик по кнопке \"Личный кабинет\"\n        login_page = LoginPage(browser, LOGIN_URL)  # Инициализация страницы логина\n        login_page.enter_user_data(create_user_data[\"email\"], create_user_data[\"password\"])  # Ввод данных для логина\n>       login_page.click_login_button()\n\nconftest.py:86: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\login_page.py:30: in click_login_button\n    self.find_element(MainPageLocator.TEXT_BURGER)\npages\\base_page.py:34: in find_element\n    return WebDriverWait(self.driver, time).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"4659ec91-39bf-41e5-96c9-872944db68ab\")>, method = <function visibility_of_element_located.<locals>._predicate at 0x000001DB3566D260>\nmessage = 'Не удалось найти элемент [\\'xpath\\', \".//h1[contains(text(), \\'Соберите бургер\\')]\"]'\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Не удалось найти элемент ['xpath', \".//h1[contains(text(), 'Соберите бургер')]\"]\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:197:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:527:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\nD:\\REPO\\Python\\Yandex\\Sunshine\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "parameters": [{"name": "browser", "value": "'firefox'"}], "start": 1739853685806, "stop": 1739853685806, "uuid": "78c627f3-6885-45b0-ae52-f3c332017cfc", "historyId": "d892298bf6f6303c23e0ca1015123823", "testCaseId": "1bd35f2a74dbe29d22e04bcd77b08985", "fullName": "tests.test_feed_page.TestFeedPage#test_display_orders_from_order_history_in_order_list", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_feed_page"}, {"name": "subSuite", "value": "TestFeedPage"}, {"name": "host", "value": "DESKTOP-1H286BJ"}, {"name": "thread", "value": "28056-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_feed_page"}]}